integrationtests/security-it/src/test/java/org/infinispan/test/integration/security/embedded/KrbLdapAuthenticationIT.java

```diff
@@ -54,21 +65,21 @@ public static WebArchive getDeployment() {
 
    public Map<String, AuthorizationPermission[]> getRolePermissionMap() {
       Map<String, AuthorizationPermission[]> roles = new HashMap<String, AuthorizationPermission[]>();
-      roles.put(ADMIN_ROLE, new AuthorizationPermission[] { AuthorizationPermission.ALL });
-      roles.put(WRITER_ROLE, new AuthorizationPermission[] { AuthorizationPermission.WRITE });
-      roles.put(READER_ROLE, new AuthorizationPermission[] { AuthorizationPermission.READ });
-      roles.put(UNPRIVILEGED_ROLE, new AuthorizationPermission[] { AuthorizationPermission.NONE });
+      roles.put(ADMIN_ROLE, new AuthorizationPermission[]{AuthorizationPermission.ALL});
```

Comment:
Is your IDE code style set properly? https://github.com/infinispan/infinispan/tree/master/ide-settings.  There are quite a few whitespace changes that shouldn't be done in this file.